antora.yml

```diff
@@ -1,5 +1,5 @@
-name: bici
-title: Bonita ICI
+name: labs
+title: Labs
 version: latest
```

Comment:
If we are sure we are only have a single version for `labs` (I have the feeling that this is the case here), we should use the Antora `no version` feature (https://docs.antora.org/antora/2.3/component-with-no-version/).
It avoids having the latest token in the url and make it explicit that there is no version.
We can still use whatever we want as `display_version`

Reply:
ok, I'm changing it according to the doc. Indeed, there will be no version.

Reply:
In the bottom left menu that let the user select the documentation component and verison, as it already exists.
When it is expanded, it looks like
![image](https://user-images.githubusercontent.com/27200110/109618880-62266300-7b38-11eb-8ee5-42434768541e.png)

Reply:
ok, I thought it was only "name" (labs). Thx.

Reply:
fixed

Reply:
As discussed we must keep a version as our current search integration requires to have a version.

Reply:
ok, fixed.